keyboards/ut472/keymaps/stefanopace/config.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include "config_common.h"
```

Comment:
This shouldn't be needed. 
```suggestion
```

Reply:
removed, the keymap still compile and works